pr_insight/git_providers/github_provider.py

```
@@ -330,11 +347,9 @@ def _publish_inline_comments_fallback_with_verification(self, comments: list[dic
             for comment in fixed_comments_as_one_liner:
                 try:
                     self.publish_inline_comments([comment], disable_fallback=True)
-                    if get_settings().config.verbosity_level >= 2:
-                        get_logger().info(f"Published invalid comment as a single line comment: {comment}")
+                    get_logger().info(f"Published invalid comment as a single line comment: {comment}")
                 except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)